Added DatabaseConnection class and modified all stubs to pull from database

rndoherty opened the pull request:
Added DatabaseConnection class and modified all stubs to pull from
database. Added litter-event-collection resource.

Commit:
Added DatabaseConnection class and modified all stubs to pull from database

Added DatabaseConnection class and modified all stubs to pull from
database. Added litter-event-collection resource.

Commit:
Removed all ID setters

Commit:
Added new user input

added ability to add a new user/volunteer
to add a new volunteer, send JSON data to
http://localhost:8080/litter-service-webapp/webapi/volunteer-list/create
/

Commit:
Added functionality to update or insert new litter event collection records

Added functionality to update or insert new litter event collection
records
test with
http://localhost:8080/litter-service-webapp/webapi/litter-event-collecti
on-list/test/
pass json data to
http://localhost:8080/litter-service-webapp/webapi/litter-event-collecti
on-list/create/

Commit:
Volunteer now returns one object when new

beastie29a commented on src/main/java/com/mattbozelka/repository/DatabaseConnection.java:
```diff
@@ -0,0 +1,231 @@
+		
+			try {
+
+				conn = DriverManager.getConnection("jdbc:mysql://localhost:3306/cleanupstars","root","password1");
```
We'll want to figure out how to utilize username/passwords on the server.
@captainMattador had some ideas for this.

rndoherty replied:
Definitely

Commit:
Added XMLElement to ID field getters

Commit:
Added user lookup and verification

User email and password can now be sent to /login/(email)/(password)/
for login

Commit:
combined litter-event-collection and litter-collection classes and methods

combined litter-event-collection and litter-collection classes and
methods
Added ability to update/create and retrieve litter-collection by userid
and eventid
path is: /litter-collection/list/(userid)/(eventId)/
